resource/resource.go

```diff
@@ -230,15 +232,7 @@ func (r *Resource) Out(
 	return dummyVersion, metadata, nil
 }
 
-func stringify1(xs map[string]interface{}) string {
-	var bld strings.Builder
-	for k, v := range xs {
-		fmt.Fprintf(&bld, "  %s: %v\n", k, v)
-	}
-	return bld.String()
-}
-
-func stringify2(xs map[string]string) string {
+func stringify[T any](xs map[string]T) string {
```

Comment:
generics :) new feature in 1.18?

Reply:
Yes, generics is the biggest feature of 1.18.
This case (stringify) is particularly simple, T is never used.